tests/integration/annotation_import/test_conversation_import.py

```diff
@@ -0,0 +1,43 @@
+import uuid
+from labelbox.data.annotation_types.annotation import ObjectAnnotation
+from labelbox.data.annotation_types.label import Label
+from labelbox.data.annotation_types.data.text import TextData
+from labelbox.data.annotation_types.ner import ConversationEntity
+
+from labelbox.schema.annotation_import import MALPredictionImport
+
+
+def test_conversation_entity(client, configured_project_without_data_rows,
+                             dataset_conversation_entity, rand_gen):
+
+    conversation_entity_annotation = ConversationEntity(name="named-entity",
+                                                        start=0,
+                                                        end=8,
+                                                        message_id="4")
+
+    entities_annotation = ObjectAnnotation(name="named-entity",
+                                           value=conversation_entity_annotation)
+
+    labels = []
+    _, data_row_uids = dataset_conversation_entity
+    configured_project_without_data_rows.create_batch(
+        rand_gen(str),
+        data_row_uids,  # sample of data row objects
+        5  # priority between 1(Highest) - 5(lowest)
+    )
+
+    for data_row_uid in data_row_uids:
+        labels.append(
+            Label(data=TextData(uid=data_row_uid),
+                  annotations=[
+                      entities_annotation,
+                  ]))
+
+    import_annotations = MALPredictionImport.create_from_objects(
+        client=client,
+        project_id=configured_project_without_data_rows.uid,
+        name=f"import {str(uuid.uuid4())}",
+        predictions=labels)
+    import_annotations.wait_until_done()
+
+    assert import_annotations.errors == []
```

Comment:
It might be a good idea to check that we can retrieve the newly created labels. For video annotations, I saw that there are no import errors but the label does not exist.

Reply:
k